client/src/pages/api/hello.ts

```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+export default async function handler(req, res) {
+	res.status(200).json({ name: "John Doe" });
+}
```

Comment:
I was trying to hook up the graphQL client to the server + NextJs but I got into a rabbit hole, so I decided to leave this one as is and try in a different PR if that hasn't been addressed yet.